_includes/payment-order-purchase.md

```diff
@@ -126,6 +126,10 @@ Content-Type: application/json
 |          | └─➔&nbsp;`subsite`                | `String(40)` | The subsite field can be used to perform [split settlement][split-settlement] on the payment. The subsites must be resolved with Swedbank Pay [reconciliation][settlement-and-reconciliation] before being used.                                                                                                                                                                                              |
 |          | └➔&nbsp;`payer`                   | `object`     | The `payer` object containing information about the payer relevant for the payment order.                                                                                                                                                                                                                                                                                                                     |
 |   ︎︎︎    | └─➔&nbsp;`consumerProfileRef`     | `string`     | The consumer profile reference as obtained through [initiating a consumer session][initiate-consumer-session].                                                                                                                                                                                                                                                                                                |
+|          | └─➔&nbsp;`email`            | `string`      | The e-mail address of the payer.                                                                                                                                                             |
+|          | └─➔&nbsp;`msisdn`            | `string`      | Number will be prefilled on payment page, if valid. The mobile number must have a country code prefix and be 8 to 15 digits in length.                                                                                                                                                             |
+|          | └─➔&nbsp;`workPhoneNumber`            | `string`      | Optional (increased chance for frictionless flow if set).                                                                                    |
```

Comment:
Is this correct information about `workPhoneNumber`? I don't see any mention of it in DX-802.